fuzz/fuzzers/server.rs

```diff
@@ -65,3 +56,37 @@ fn fuzz_acceptor_api(data: &[u8]) {
         }
     }
 }
+
+fn fuzz_accepted(stream: &mut dyn io::Read, accepted: Accepted) {
+    let maybe_server = accepted.into_connection(Arc::new(
+        ServerConfig::builder_with_provider(rustls_fuzzing_provider::provider().into())
+            .with_safe_default_protocol_versions()
+            .unwrap()
+            .with_no_client_auth()
+            .with_cert_resolver(rustls_fuzzing_provider::server_cert_resolver()),
+    ));
+
+    let mut server = match maybe_server {
+        Ok(conn) => conn,
+        Err(_) => return,
+    };
+
+    service_connection(stream, &mut server);
```

Comment:
Nit: in this case I think the `service_connection()` call fits on a single line within the `match` arm? Don't see much reason to break it out.